api.go

```diff
@@ -1 +1,149 @@
 package main
+
+import (
+	"encoding/json"
+	"io"
+	"net/http"
+	"time"
+)
+
+type API struct {
+	core *Core
+	Lg   *slog.Logger
```

Comment:
Логгер тоже не экспортируемый должен быть

```suggestion
	lg   *slog.Logger
```